mmdet/models/detectors/test_mixins.py

```diff
@@ -18,9 +18,15 @@ def aug_test_rpn(self, feats, img_metas, rpn_test_cfg):
             for i, proposals in enumerate(proposal_list):
                 aug_proposals[i].append(proposals)
         # after merging, proposals will be rescaled to the original image size
+        _img_metas = []
```

Comment:
Thanks a lot for this pr! 
A comment before this line is needed to briefly explain why we need to reorganize the order of img_metas. 
'_img_meta' and '_img_metas' can be renamed as 'aug_img_meta' and 'aug_img_metas' to be consistent with 'aug_proposals'.